### DISCUSSION TITLE
Feature Request:  an OR logic with columns(*) function

### DISCUSSION THREAD

**Discussion by RichieZ** [duckdb/duckdb#12826] [Ideas]
while "WHERE COLUMNS(*) > 1; " is equivalent to: x > 1 AND y > 1 AND z > 1
How do i make  an equivalent  of " x > 1 OR y > 1 OR z > 1 " by columns(*) function?
By this function, i could use to find if any column has a specific value.

**Comment by soerenwolfers** [comment]
A very roundabout solution would be to use the [`EXCEPT`]( https://duckdb.org/docs/sql/query_syntax/setops.html#:~:text=7-,EXCEPT,occur%20in%20the%20left%20query.) clause to remove the negation of the condition you're interested in (i.e. use de Morgan's law)

**Comment by soerenwolfers** [comment]
@RichieZ this is now possible, see https://github.com/duckdb/duckdb/issues/15676
